packages/expo-modules-core/common/cpp/fabric/ExpoViewProps.cpp

```diff
@@ -7,12 +7,27 @@ namespace react = facebook::react;
 
 namespace expo {
 
+/**
+ Borrows the props map from the source props and applies the update given in the raw props.
+ */
+std::unordered_map<std::string, folly::dynamic> propsMapFromProps(const ExpoViewProps &sourceProps, const react::RawProps &rawProps) {
+  // Move the contents of the source props map – the source props instance will not be used anymore.
+  std::unordered_map<std::string, folly::dynamic> propsMap = std::move(sourceProps.propsMap);
```

Comment:
@Kudo do you think the move here is safe? As far as I know, the source props will not be used anymore and the `propsMap` is used only once per the lifetime of the `ExpoViewProps` instance 🤔 We could do a copy if you're not sure.

Reply:
at first glance, it's strange to me because there's `const ExpoViewProps &sourceProps` in the function declaration and we won't like to mutate the sourceProps. i think it's from react-native to pass `sourceProps` as a const reference. however, since the `propsMaps` is controlled by us. that's somehow reasonable. let's keep it as-is.